operator-framework-junit5/src/main/java/io/javaoperatorsdk/operator/junit/LocallyRunOperatorExtension.java

```diff
@@ -43,6 +43,8 @@ public class LocallyRunOperatorExtension extends AbstractOperatorExtension {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(LocallyRunOperatorExtension.class);
   private static final int CRD_DELETE_TIMEOUT = 1000;
+  private static final LinkedList<AppliedCRD> appliedCRDs = new LinkedList<>();
```

Comment:
What is the reason using LinkedList here? Just since those are large objects?

Reply:
LinkedList is only needed to keep FIFO ordering of applied CRDs, but I don't think it's necessary. Why large? AppliedCRD is only two Strings - https://github.com/operator-framework/java-operator-sdk/blob/main/operator-framework-junit5/src/main/java/io/javaoperatorsdk/operator/junit/LocallyRunOperatorExtension.java#L333

Reply:
The order should not be relevant for CRDs, so it is rather confusing. I would lean more to just have a List.